app/qml/ProjectWizardPage.qml

```diff
@@ -0,0 +1,203 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.3
+import QtGraphicalEffects 1.0
+import QtQuick.Dialogs 1.2
+import QgsQuick 0.1 as QgsQuick
+import lc 1.0
+import "."
+import "./components" // import InputStyle singleton
+Item {
+  id: projectWizardPanel
+
+  signal back
+
+  property real rowHeight: InputStyle.rowHeight
+  property var fontColor: InputStyle.fontColor
+  property var bgColor: InputStyle.clrPanelMain
+  property real panelMargin: 10 * QgsQuick.Utils.dp
+
+  property ListModel items: ListModel {}
+  property var supportedTypes: {
+    var types = __fieldsModel.supportedTypes()
```

Comment:
I do not quite understand this part could you please add comment here what is it doing? I understand you are trying to convert QVariantMap and fill the ListModel with it however, is the `items` property used somewhere? Is it implicitly converting something?